helpers/util.sh

```diff
@@ -149,7 +149,7 @@ yesnoall() {
        true
        ;;
     [aA]*)
-       BUILDALL=y
```

Comment:
This means BUILDALL (declared at the top of the file) is no longer used anywhere anymore